pyparsing/core.py

```diff
@@ -300,7 +306,7 @@ def inlineLiteralsUsing(cls):
 
     def __init__(self, savelist=False):
         self.parseAction = list()
-        self.failAction = None
+        self.failAction = noop
```

Comment:
ensure `failAction` is always a function, then we need not do a type check (see below)